src/components/views/read/cvForm/Review.tsx

```diff
@@ -109,32 +59,33 @@ const Review: FC<Props> = ({
     prevPage();
     prevStep();
   };
-  const [cards, setCards] = useState([{ id: 1 }]);
-  const handleDownload = (): void => {
-    const CvGenerated = 'link';
-    <a
-      href={CvGenerated}
-      download="Example-PDF-document"
-      target="_blank"
-      rel="noreferrer"
-    >
-      <Button type="button">Download .pdf file</Button>
-    </a>;
-  };
-  console.log(values);
-  // const personalData = values['Personal Info'];
-  // const educationData = values.Education;
-  // const workData = values['Work Experience'];
-  // const skillsData = values.Skills;
-  // const portfolioData = values.Portfolio;
-  // const motivationData = values['Motivation Letter'];
-  // const referencesData = values.References;
-
+  console.log(cvValues);
   return (
-    <div>
-      <h2>Review</h2>
+    <Box>
+      <Typography>Generated CV</Typography>
+      <Box justifyContent="center" display="flex">
+        {templateData[0].selected === true &&
+          templateData[0].title === 'professional template' && (
```

Comment:
This is a case where as we only got one template, I can directly access it from the array and apply the approach about checking the selection boolean value, but when we add more templates, it will be dynamically checks which template have the true value, but at that moment also, if we got the template which got true selection value, and as I'm doing here the template will be a separate file like `FirstTemplate` file, so there might be others for other template, then how will I handle that? that thing I have thought about, and for now put it as fixed indexing.